.github/workflows/tests.yaml

```diff
@@ -32,8 +32,8 @@ jobs:
         lfs: true
 
     # Install emcpy
-    - name: Upgrade pip
-      run: $CONDA/bin/pip3 install --upgrade pip
+    #- name: Upgrade pip
+    #  run: $CONDA/bin/pip3 install --upgrade pip
```

Comment:
Should these commented lines be removed?

Reply:
@EdwardSafford-NOAA This is how @CoryMartin-NOAA  solved the EVA troubleshooting issues last week so we will have to figure out if this is a temporary fix or the right fix.

Reply:
Yeah I don't know if it is permanent or not to not run these.  We can remove and add back if you prefer, it will be tracked in git.